examples/kaniko/skaffold.yaml

```diff
@@ -8,6 +8,9 @@ build:
   cluster:
     pullSecretName: e2esecret
     namespace: default
+#    annotations:
```

Comment:
Would you mind indenting this and the tolerations to align with the `namespace:` elements?

Reply:
Thank you so much for checking in detail. 👍   I aligned both of them with namespace.